.github/ISSUE_TEMPLATE/bugs-performance-issues.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report (including performance and build issues)
+about: Help us improve the GPflow code base
```

Comment:
> Help us improve the GPflow code base
This is true of most ticket types, right. Can we say something more specific, like
```suggestion
about: Help us find mistakes in the GPflow code base
```